k9mail/src/main/res/values/arrays.xml

```diff
@@ -512,6 +512,18 @@
         <item>0</item>
     </string-array>
 
+    <string-array name="account_settings_attachment_resize_factor_entries">
+        <item >@string/account_settings_attachments_resize_factor_entry_original</item>
+        <item >@string/account_settings_attachments_resize_factor_entry_half</item>
+        <item >@string/account_settings_attachments_resize_factor_entry_one_fourth</item>
+    </string-array>
+
+    <string-array name="account_settings_attachment_resize_factor_values">
+        <item>1</item>
```

Comment:
Instead of an int this could be an enum - similar to how `global_settings_notification_hide_subject_values` behaves below.